app/views/transactions/_form.html.slim

```diff
@@ -6,7 +6,7 @@
 
   - if f.object.category.try(:system?)
     = f.input :category, disabled: true
-  - elsif params[:action] == 'new' || params[:action] == 'create'
+  - elsif params[:action] == ('new' || 'create')
```

Comment:
@yunixon it's a wrong code. If you want to refactor it - you can use `['new', 'create'].include?(params[:action])`. Please check specs, cos I think they are wrong too, if there are no failed specs with this code.